### DISCUSSION TITLE
How to install SEB for multiple LMS vendors on single workstations?

### DISCUSSION THREAD

**Discussion by Rokartt4209** [SafeExamBrowser/seb-win-refactoring#560] [General]
I need to use SEB on student computers for two different LMS vendors.
Each vendor has their own URL for student logins.  How can I set up a desktop shortcut for each vendor?  
Can SEB take parameters to indicate which configuration file to read and which URL to navigate to or will SEB need to be installed into separate folders for the two vendors?

**Comment by dbuechel** [comment]
You can e.g. create two different configuration files, where each file would point to the respective LMS instance. You could also create two Windows application shortcuts with the URL as parameter. All required information about the configuration and usage of SEB can be found here: https://safeexambrowser.org/windows/win_usermanual_en.html.
